packages/test-e2e/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@hwp/test-e2e",
```

Comment:
We might not need this for now so we can remove the whole package to make it easier to review the app